packages/node_modules/@ciscospark/react-component-spark-fonts/src/index.js

```diff
@@ -1,3 +1 @@
-import styles from './fonts.css';
-
-export default styles;
+import './fonts.scss';
```

Comment:
Does this break anything in the space widget using this?

Reply:
Not that I'm aware of.